index.md

```diff
@@ -73,7 +75,7 @@ I use this method along with…
 
 #### Asana
 
-I was introduced to [Asana](https://asana.com/) by a [friend](https://twitter.com/neuroamanda) around a year ago, and have been using it on and off for long-term planning of my activities.
+I was introduced to [Asana](https://asana.com/) by a [friend](https://twitter.com/neuroamanda) and have been using it on and off for long-term planning of my activities.
 As a PhD student, you will often need to plan well into the future, with many external deadlines looming ahead: progress reports, vivas, conference submissions etc.
```

Comment:
You make a fair point about potential hindrances, but I'm not sure I want to address it (nor the collaborative aspects). Your supervisors may have their own methods and might not want to use new/different tech, so this is largely advice for individuals.